src/ui/windows/TogglDesktop/TogglDesktop/ui/views/Timeline.xaml

```diff
@@ -37,15 +37,15 @@
                       IsChecked="{Binding RecordActivity}">
                     Record activity
                 </mah:ToggleSwitch>
-                <Grid Width="16" Height="16"
-                      VerticalAlignment="Center"
+                <Grid VerticalAlignment="Bottom"
                       HorizontalAlignment="Left"
-                      Margin="5 12 0 0">
-                    <Canvas MouseEnter="RecordActivityInfoBoxOnMouseEnter" x:Name="InfoIcon">
-                        <Path Fill="#B1B1B5" Data="{StaticResource Toggl.InfoIconGeometry}"/>
-                    </Canvas>
-                    <Popup IsOpen="False" Name="RecordActivityInfoPopup" StaysOpen="False" UseLayoutRounding="True"
-                           PlacementTarget="{Binding ElementName=InfoIcon}" AllowsTransparency="True">
+                      Margin="5 0 0 2">
+                    <ToggleButton Name="InfoButton"
+                                  Style="{StaticResource Toggl.Timeline.InfoButton}"
+                                  MouseEnter="OnInfoButtonMouseEnter"

```

Comment:
It's easier to do by binding the popup's `StaysOpen` property to button's `IsMouseOver`:
```StaysOpen="{Binding ElementName=InfoButton, Path=IsMouseOver, Mode=OneWay}"```.

I also think it might be more reliable than MouseEnter/MouseLeave - during the recent PR review of TE tooltip, there has been an unexpected dragging tooltip effect, possibly connected to the MouseEnter/MouseLeave events.

Reply:
Fixed